config/routes.rb

```diff
@@ -429,6 +429,8 @@ def external_redirect(path_prefix, target)
 
   get "/government/uploads/system/uploads/attachment_data/file/:id/*file.:extension/preview" => "csv_preview#show", as: :csv_preview
 
+  resources :document_list_export_request, path: "/export/:document_type_slug", param: :export_id, only: [:show]
```

Comment:
This adds a new route, but I wouldn't have guessed a new route was being added by the commit message contents. Could you spend a bit of time explaining what's being added and why?

Reply:
I've added an explanation in the relevant commit message.

Reply:
I'm still not seeing an explanation in https://github.com/alphagov/whitehall/pull/5709/commits/dfe36a60b5c355e7a64005a192ab3f5300d1df5f - did you add it to the right commit?

Reply:
It's in d9af9e765afe068edca914566feed0b47711b56b — that's where the controller itself is added. Actually the route should have been added there too, separating it out then committing it at the same time as the tests wasn't really what I had in mind.

Reply:
From what you're saying, it sounds like this line should be moved to the first commit, but that it might be more work than it's worth. I'll leave it up to you to decide.